modele-social/règles/dirigeant.yaml

```diff
@@ -954,7 +954,31 @@ dirigeant . indépendant . cotisations et contributions . début activité . ass
   références:
     Fiche Urssaf: https://www.urssaf.fr/portail/home/independant/mes-cotisations/les-etapes-de-calcul/le-mode-de-calcul/les-cotisations-provisionnelles/debut-dactivite.html
 
-    
+dirigeant . indépendant . cotisations et contributions . après début:
+  titre: Régularisations des cotisations à partir de la troisième année d'activité
```

Comment:
```suggestion
  titre: Régularisations des cotisations en régime de croisière
```

Reply:
Effectivement... Ok pour enlever partout la référence à un "régime de croisière". Ceci étant dit "à partir de la troisième année" n'est pas techniquement correct non plus, puisque la régularisation commence courant de la 2eme année il me semble.

Peut-être qu'on pourrait renommer en "Cotisations provisionnelle et régularisation" ou "Comment fonctionne la régularisation des cotisations ?" ou encore "Qu'est-ce que la régularisation des cotisations provisionnelle ?" (une question est plutôt adaptée pour le référencement)

Reply:
Ok pour la reformulation. En revanche c'est bien à partir de l'an 3 que c'est le régime "basé sur n-2"

Reply:
Mmmh, j'avais compris que cela arrivait courant de l'année n+2 (vers juin), une fois que la déclaration de revenu traitée par l'Urssaf. Par exemple, pour une entreprise crée en janvier 2020, cela donne courant 2021. Mais je peux me tromper.

Reply:
En effet, tu dis vrai sur la partie "régularisation". Pour ma part, je me concentrais sur la considération "base de calcul n-2" qui est orthogonale. En fait il y a 2 notions:

- le provisionnel (ou base de calcul), qui a deux régimes n ≤ 2 et n ≥ 3
    - n ≤ 2: provisionnel = forfaitaire
    - n ≥ 3: provisionnel = n - 2
    - dans tous les cas: il est possible de définir un provisionnel custom
- la régularisation qui arrive (bien entendu) pour toutes les années:
    - à l'année n, on paie le `provisionnel` (alors que le réel peut être noté `réel`, pas encore connu de l'administration)
    - à l'année n+1, la déclaration faite (sans doute à partir de en juin en effet comme tu dis), la régularisation est effectuée: on compense en payant (ou étant remboursé de)`réel - provisionnel`.

Vu comment j'ai reformulé le `après début`, c'est bien la notion de "régularisation" qui est traitée par ce texte. Tu as donc raison de faire la remarque: il faut que je vire le `non applicable si` car ce texte doit apparaître dans tous les cas, peu importe le régime.

Ce qui m'embête en revanche, c'est qu'il manque un texte explicitant le régime de provisions `n ≥ 3`. Je vais faire une PR pour essayer rajouter ça et on voit si c'est raisonnable ou non en termes de texte.

Reply:
Et pour être tatillons, en fait il y a 3 régimes de provisionnels:

- n = 1: provisionnel = forfaitaire, non mis à jour
- n = 2: provisionnel = forfaitaire, puis après déclaration provisionnel = CA(n=1)
- n ≥ 3: provisionnel = CA(n=n - 2) puis après déclaration provisionnel = CA(n=n-1)

Reply:
> Vu comment j'ai reformulé le après début, c'est bien la notion de "régularisation" qui est traitée par ce texte. Tu as donc raison de faire la remarque: il faut que je vire le non applicable si car ce texte doit apparaître dans tous les cas, peu importe le régime.

Dans l'idée, oui, mais pour éviter un trop grand nombre de popup bleu, je me disais qu'on pouvais parler à la fois de la régularisation et des cotisations provisionnelles si n <= 2 ans (un seul bloc bleu).

Donc je laisserait tel quel :)

Reply:
D'accord pour la partie UI, j'ai gardé qu'un seul bloc bleu.

En revanche j'ai repris la règle et ai notamment retiré le `non appliable si`, car au niveau logique ce n'était pas cohérent (la remarque sur la régularisation est toujours valable). J'ai du jouer un peu avec le composant Condition pour y arriver, cependant. Je crois d'ailleurs que quelque chose me gène avec ce composant, à voir…

Reply:
@johangirod ok pour toi sur mon 2e commit? J'ai préparé une autre PR pour reprendre le "problème" du composant Condition, et donc améliorer un peu ce que j'ai mis dedans.